data/crac-file/crac-io-json/src/main/resources/CracSchema.json

```diff
@@ -8,12 +8,46 @@
     "name" : {
       "type" : "string"
     },
+    "contingencies" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "id" : "urn:jsonschema:com:farao_community:farao:data:crac_api:Contingency",
+        "properties" : {
+          "networkElements" : {
+            "type" : "array",
+            "items" : {
+              "type" : "object",
+              "id" : "urn:jsonschema:com:farao_community:farao:data:crac_api:NetworkElement",
+              "properties" : {
+                "id" : {
+                  "type" : "string"
+                },
+                "name" : {
+                  "type" : "string"
+                }
+              }
+            }
+          },
+          "name" : {
+            "type" : "string"
+          },
+          "id" : {
+            "type" : "string"
+          }
+        }
+      }
+    },
     "cnecs" : {
       "type" : "array",
       "items" : {
         "type" : "object",
         "id" : "urn:jsonschema:com:farao_community:farao:data:crac_api:Cnec",
         "properties" : {
+          "criticalNetworkElement" : {
+            "type" : "object",
+            "$ref" : "urn:jsonschema:com:farao_community:farao:data:crac_api:NetworkElement"
+          },
           "state" : {
```

Comment:
Do you think that `states` could be at the "root" of the JSON-crac (like preventive-states), as the function `getStates()` now exists in the crac-api/Crac ?

Json-Cnec would then refer to states-id, but not to the full content of the states. 

I would, in my opinion, give a more compact and readable JSON crac.

Reply:
To be reported in technical debt... Might evolve quickly with next crac refactoring.

Reply:
okay